package.json

```diff
@@ -12,8 +12,9 @@
     "lint:fix": "prettier --write 'src/**/*.{js,json,jsx}'"
   },
   "dependencies": {
+    "ajv": "^6.10.0",
```

Comment:
Not sure whether updates were necessary in both top level and functions directory `package.json` - perhaps from doing `firebase deploy ---only functions`?